mkdocs.yml

```diff
@@ -45,11 +45,15 @@ nav:
   - 'Obtaining DCR': 'getting-started/obtaining-dcr.md'
   - 'Using the Block Explorer': 'getting-started/using-the-block-explorer.md'
 - Governance:
+  - Politeia:
+    - 'Politeia': 'governance/politeia.md'
+    - 'Proposal Guidelines': 'governance/politeia/proposal-guidelines.md'
+    - 'Example Proposals': 'governance/politeia/example-proposals.md'
   - 'Introduction to Decred governance': 'governance/introduction-to-decred-governance.md'
   - 'Consensus Rules Voting': 'governance/consensus-rules-voting.md'
   - 'How to Vote on Consensus Rule Changes': 'governance/how-to-vote.md'
-  - 'Politeia Proposals and Voting': 'governance/politeia.md'
   - 'Decred Constitution': 'governance/decred-constitution.md'
+
```

Comment:
Accidental newline?

Reply:
Yep. Removed. Double checked that menus are still rendering correctly.